README.md

```diff
@@ -16,6 +16,9 @@ and [building easy-to-read messages ready to be sent to the scanned organization
 
 If you want to use additional modules that weren't included here due to non-BSD-compatible licenses, browse to the [Artemis-modules-extra](https://github.com/CERT-Polska/Artemis-modules-extra) repository.
 
+If you work as a developer and you want to modify code, you have to do some modifications ⮕  **[Developers](#develop)**
```

Comment:
can all these changes be put in one place? e.g. ./scripts/start_develop that runs scripts/start with some tweaks?

Reply:
Maybe create docker-compose.prod.yaml and Dockerfile.prod and put as a variable in scritps/start_develop?